test/integration/smoke/test_ssvm.py

```diff
@@ -1197,3 +1205,148 @@ def test_10_destroy_cpvm(self):
         # Call to verify cloud process is running
         self.test_04_cpvm_internals()
         return
+
+    @attr(
+        tags=[
+            "advanced",
+            "advancedns",
+            "smoke",
+            "basic",
+            "sg"],
+        required_hardware="true")
+    def test_11_ss_nfs_version_on_ssvm(self):
+        """Test NFS Version on Secondary Storage mounted properly on SSVM
+        """
+
+	# 1) List SSVM in zone
+	# 2) Get id and url from mounted nfs store
+	# 3) Update NFS version for previous image store
+	# 4) Stop SSVM
+	# 5) Check NFS version of mounted nfs store after SSVM starts 
+
+	nfs_version = self.config.nfsVersion
+	if nfs_version == None:
+	    self.skipTest('No NFS version provided in test data')
+
+	#List SSVM for zone id
+        list_ssvm_response = list_ssvms(
+            self.apiclient,
+            systemvmtype='secondarystoragevm',
+            state='Running',
+            zoneid=self.zone.id
+        )
+        self.assertEqual(
+            isinstance(list_ssvm_response, list),
+            True,
+            "Check list response returns a valid list"
+        )
+        self.assertEqual(
+            len(list_ssvm_response),
+            1,
+            "Check list System VMs response"
+        )
+
+        ssvm = list_ssvm_response[0]
+        image_stores_response = ImageStore.list(self.apiclient,zoneid=self.zone.id)
+
+	if self.hypervisor.lower() in ('vmware', 'hyperv'):
+            # SSH into SSVMs is done via management server for Vmware and Hyper-V
+            result = get_process_status(
+                self.apiclient.connection.mgtSvr,
+                22,
+                self.apiclient.connection.user,
+                self.apiclient.connection.passwd,
+                ssvm.privateip,
+                "mount | grep 'type nfs'",
+                hypervisor=self.hypervisor)
+
+	for res in result:
+            split_res = res.split(" on ")
```

Comment:
Relying on the result having only one space seems brittle.  Please consider splitting on "on" and trimming the results or using a regular expression.